src/lib/input/input-container.spec.ts

```diff
@@ -1020,6 +1136,29 @@ class MdInputContainerWithFormErrorMessages {
 
 @Component({
   template: `
+    <form #form="ngForm" novalidate>
```

Comment:
Remove `novalidate`? This is added by default by the `NgNoValidate` directive in `@angular/forms`.